app/middleware/renderer.js

```diff
@@ -1,8 +1,8 @@
 const log = require('../lib/logger');
 const mapLink = require('../lib/mapLink');
 
-function notFound(req, res, msg) {
-  log.warn(msg, 'Not found (404)');
+function notFound(req, res) {
+  log.warn({ req }, 'Not found (404)');
```

Comment:
Just a comment - @adamgraves-choices had concerns about us logging the whole of the request object. I did look to see if there was a sub-set of the object which could be logged but wasn't successful as I wasn't sure what information you need to collect in order to diagnose, say, a not found error.

I think my preference would be still to log the whole request.